README.md

```diff
@@ -0,0 +1,85 @@
+<p align="center">
+    <img src=".github/paypal-php-sdk.svg" width="200" alt="PayPal Logo" />
+</p>
+
+<p align="center">
+    <h1 align="center">PayPal SDK for PHP</h1>
+</p>
+
+A Paypal PHP SDK that allows integration of PayPal services into PHP applications.
+This SDK provides a simple and easy-to-use interface for working with PayPal's REST APIs, making it easier to accept payments, handle PayPal-related tasks and errors.
+
+> [!WARNING]
+> Currently this SDK is an extraction of the implementation within Shopware's [PayPal plugin](https://github.com/shopware/SwagPayPal).
+> Only use cases and API responses used within the plugin are covered.
+
+## Quick start
+### Installation
+You can install the SDK via Composer. Run the following command in your terminal:
+
+```bash
+composer require shopware/paypal-sdk
+```
+
+In addition, a PSR client implementation is needed. We recommend [Guzzle HTTP](https://github.com/guzzle/guzzle) or Symfony's [HttpClient](https://symfony.com/doc/current/http_client.html).                                               
+```bash
+composer require guzzlehttp/psr7 php-http/guzzle7-adapter
```

Comment:
Add these to the suggest section of the composer.json

https://getcomposer.org/doc/04-schema.md#suggest